pipelines/DECam/SingleFrame.yaml

```
@@ -0,0 +1,23 @@
+description: >-
+  Single-frame pipeline for the case in which
+  no templates exist, specialized for DECam
+instrument: lsst.obs.decam.DarkEnergyCamera
+imports:
+  - location: $AP_PIPE_DIR/pipelines/DECam/ProcessCcd.yaml
+    include: 
+      - processCcd
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/SingleFrame.yaml
+    include:
+      - getRegionTimeFromVisit
+      - mpSkyEphemerisQuery
+      - ssSingleFrameAssociation
+      - initialPviCore
```

The following might be more robust to pipeline changes, but hopefully it becomes irrelevant when we remove `ProcessCcd.yaml`:
```suggestion
    exclude:
      - isr
      - calibrateImage
```